examples/pn532_readwrite_mifare.py

```diff
@@ -1,26 +1,8 @@
+# SPDX-FileCopyrightText: 2015 Tony DiCola, Roberto Laricchia, for Adafruit Industries
+# SPDX-FileCopyrightText: Francesco Crisafulli
```

Comment:
I originally tried having all these names on one line but pylint didn't like the length of that line.

I split it into two `SPDX-FileCopyrightText` lines, but I am not positive if that is really the correct solution.

I'll leave this one un-merged until that can be confirmed.